zkevm-circuits/src/bytecode_circuit/circuit.rs

```diff
@@ -680,6 +690,17 @@ impl<F: Field> BytecodeCircuitConfig<F> {
         Ok(())
     }
 
+    fn annotate_circuit(&self, region: &mut Region<'_, F>) {
```

Comment:
Could you add this here?
```rust
self.bytecode_table.annotate_columns_in_region(&mut region);
self.keccak_table.annotate_columns_in_region(&mut region);
```
It will give names to the bytecode and keccak table when used in constraints in this region (the `bytecode_table.annotate_columns(meta);` only gives column names for lookups)

Reply:
Oh! I forgot this. thanks and fixed in [99cd564](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1152/commits/99cd564d5db77197f476bbb7d82a415823ad6572)